src/components/tx/GasParams/index.tsx

```diff
@@ -34,6 +34,7 @@ const GasParams = ({ params, isExecution, isEIP1559, onEdit, gasLimitError }: Ga
 
   const chain = useCurrentChain()
   const isLoading = !gasLimit || !maxFeePerGas
+  const isError = gasLimitError && !gasLimit
```

Comment:
Not sure if we need `gasLimitError` or could pass the loading flag instead and do `const isError = !gasLimitLoading && !gasLimit`